### PR TITLE
Added new Avi module to setup SSL profile.

### PR DESCRIPTION
**Pull request by grastogi23**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

network.avi.avi_sslprofile

##### ANSIBLE VERSION

```
ansible 2.3.0
  config file = /home/grastogi/.ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY

New module for setting to setup SSL Profile.

```

```

### CODE REVIEW

**Comment by khaltore** [comment]
shipit

**Comment by ericsysmin** [comment]
shipit

File: lib/ansible/modules/network/avi/avi_sslprofile.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+version_added: "2.3"
+options:
+    state:
+        description:
+            - The state that should be applied on the entity.
+        default: present
+        choices: ["absent","present"]
+    accepted_ciphers:
+        description:
+            - Ciphers suites represented as defined by http //www.openssl.org/docs/apps/ciphers.html.
```
Comment (gundalow) [inline lib/ansible/modules/network/avi/avi_sslprofile.py]: `U(http://www.openssl.org/docs/apps/ciphers.html)`

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by ericsysmin** [comment]
shipit

**Review by gundalow — APPROVED** [review]

**Pull request merged**